java/maven/cloudbuild.yaml

```diff
@@ -0,0 +1,8 @@
+# In this directory, run the following command to build this builder.
+# $ gcloud container builds submit . --config=cloudbuild.yaml
+
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: ['build', '--tag=gcr.io/$PROJECT_ID/maven', '.']
```

Comment:
Yes - mvn is the name of the CLI itself (although the `mvn` CLI runs the maven build system). We're trying to have a metaphor of the cloudbuild.yaml being "kinda like a shell script, if you squint".